### PR TITLE
[vcpkg docs] Update Table of Content and Korean Readme

### PR DESCRIPTION
**Pull request by jiyuPk**
Updated Korean Readme and deleted direction to Installing GCC for macOS before 10.15 in Table of Content

### CODE REVIEW

**Comment by PhoebeHui** [comment]
@AkiaCode, @vyujing, could you help review?

**Review by PhoebeHui — APPROVED** [review]

File: README_ko_KR.md [hunk 1]
```diff
@@ -159,10 +160,10 @@ CMake 및 CMake Tools for VSCode에서 vcpkg를 가장 잘 사용하는 방법
 다른 툴에 대해서는 [통합 가이드][getting-started:integration]을 확인하세요.
 ## 리눅스 개발자 도구 설치하기
 
-리눅스의 다양한 배포판에는 다양한 패키지가 있습니다.
-설치 필요:
+리눅스의 다양한 배포판에는 설치해야되는 다양한 패키지가 있습니다:
```
Comment (AkiaCode) [inline README_ko_KR.md:163]: `설치해야되는` -> `설치해야 되는`

**Comment by PhoebeHui** [comment]
@jiyuPk, could you address the review suggestion?

**Comment by jiyuPk** [comment]
@PhoebeHui I did.

**Review by PhoebeHui — APPROVED** [review]
Thanks for your updates!

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**